.github/workflows/debezium-workflow.yml

```diff
@@ -768,3 +781,36 @@ jobs:
           -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn         
           -Dmaven.wagon.http.pool=false 
           -Dmaven.wagon.httpconnectionManager.ttlSeconds=120          
+
+  build_ui:
+    needs: [check_style, file_changes]
+    if: ${{ needs.file_changes.outputs.debezium-ui-changed == 'true' }}
+    name: "UI"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Action (UI)
+        uses: actions/checkout@v2
+        with:
+          repository: debezium/debezium-ui
+          path: debezium-ui
+
+      - name: Set up Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Cache Maven Repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: maven-debezium-test-build-${{ hashFiles('core/**/pom.xml') }}
+          restore-keys: |
+            maven-debezium-test-build-${{ hashFiles('core/**/pom.xml') }}
+
+      - name: Build Debezium UI
+        run: >
+          ./mvnw clean install -f debezium-ui/backend/pom.xml -am -pl backend -B
+          -Ddebezium.test.records.waittime=20
+          -Dformat.formatter.goal=validate
+          -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
@jpechane I tried running with this command, but it fails. What do you think would be the right command?

Reply:
Hi @ani-sha so the issue is that the UI uses the Maven wrapper which is only available from the `./debezium-ui` path so perhaps for this build step we should use:
```
cd debezium-ui && ./mvnw clean install -f backend/pom.xml ...
```
Give that a try and see if that works.

Reply:
Another option is to add `working-directory` to the build step and specify `debezium-ui` and omit the `cd` portion of the runnable command too.

Reply:
Already tested in https://github.com/debezium/debezium/pull/3481
I am going to get the commit here
But we need to fix the UI build first
It fails for me locally too